app/src/main/java/com/github/fribourgsdp/radio/SongFragment.kt

```diff
@@ -60,6 +52,15 @@ open class SongFragment : MyFragment(R.layout.fragment_song) {
         val artistTitle : TextView = requireView().findViewById(R.id.ArtistName)
         val lyricsEditText : EditText = requireView().findViewById(R.id.editTextLyrics)
 
+        val user = User.load(requireContext())
+        playlist = user.getPlaylistWithName(playlistName)
+        song = playlist.getSong(songName)
+        initialLyrics = song.lyrics
+        currentLyrics = initialLyrics
+        if(song.lyrics == ""){
```

Comment:
Shouldn't you replace this with your initialLyrics variable?

Reply:
this is not my code but yes